frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -311,6 +311,33 @@ export class OutputArtifactLoader {
         return buildArtifactViewer(script);
       }),
     );
+    const EvaluatorArtifactUris = filterArtifactUrisByType('ModelEvaluation', artifactTypes, artifacts);
+    viewers = viewers.concat(
+      EvaluatorArtifactUris.map(uri => {
+        const configFilePath = uri + '/eval_config.json';
+        const script = [
```

Comment:
Can you put a comment to refer to https://github.com/tensorflow/model-analysis/issues/10#issuecomment-587422929 here?
That helps clear future confusion where the hack comes from

Reply:
Done